packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel.Plugin/src/Microsoft.Generator.CSharp.ClientModel.Plugin.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Generator.CSharp.ClientModel.Plugin</PackageId>
+    <OutputType>Exe</OutputType>
+    <Version>1.0.0-beta.1</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.ClientModel" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <!--Add this as a project reference for now until the core generator library is packaged and published.-->
```

Comment:
its published to private feed lets pull from there so its as close to real as possible

Reply:
This is the actual SCM plugin project.

Reply:
Collapsed this back into a single project